servers/cromwell/README.md

```diff
@@ -15,15 +15,22 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
     export CROMWELL_URL=http://192.168.0.106:8000/api/workflows/v1
     ```
 
-2. Add a file named `config.json` to `job-manager/servers/cromwell/jobs` that contains the username and password for the specified cromwell instance (You may want to skip this step if the Cromwell instance does not have HTTP Basic Authentication):
+2. (Optional) By default, the shim layer uses **5** `sync` [Gunicorn workers](http://docs.gunicorn.org/en/stable/settings.html#worker-class), the number of workers is **5**. 
+    Job Manager also comes with `gevent` workers, you can override the default worker type and use dynamic number of workers(based on the number of CPU cores) by:
+    
+    ```
+    export GUNICORN_CMD_ARGS="--workers=$((2 * $(getconf _NPROCESSORS_ONLN 2>/dev/null || getconf NPROCESSORS_ONLN 2>/dev/null || echo 2) + 1)) --worker-class gevent"
+    ```
+    
+3. Add a file named `config.json` to `job-manager/servers/cromwell/jobs` that contains the username and password for the specified cromwell instance (You may want to skip this step if the Cromwell instance does not have HTTP Basic Authentication):
     ```
     {
       "cromwell_user" : "USERNAME",
       "cromwell_password" : "PASSWORD"
     }
     ```
 
-3. (Optional) If you want to change either the predefined view or the default behavior of UI to some extent, e.g. display more columns in the job list view such as labels of the jobs, or make more columns to be popped up by the query builder, you can add a `capabilities_config.json` file to `job-manager/servers/cromwell/jobs` to override the pre-defined configurations. The `capabilities_config.json` should **strictly** follow the following structure:
+4. (Optional) If you want to change either the predefined view or the default behavior of UI to some extent, e.g. display more columns in the job list view such as labels of the jobs, or make more columns to be popped up by the query builder, you can add a `capabilities_config.json` file to `job-manager/servers/cromwell/jobs` to override the pre-defined configurations. The `capabilities_config.json` should **strictly** follow the following structure:
```

Comment:
minor nit: Does this not display correctly if we use `1.` for all of these? (I know sometimes it doesn't work but forget the circumstances)

Reply:
I think all `1.`s will definitely work, as long as we have proper indentations. I changed here before due to my OCD on raw-text :( But now since we have more numbered items, I think it is a good idea to change back to use all `1.` here. Will do it.